### PR TITLE
Update dev-packages dependencies & string format method

### PR DESCRIPTION
**Pull request by Davidy22**
Updated some things that were deprecated/discontinued, did the make update as per #1904 discussion and cleaned up the moans. Part of prep for guake 3.8

### CODE REVIEW

File: guake/__init__.py [hunk 1]
```diff
@@ -39,11 +39,7 @@ def vte_version():
 
     from gi.repository import Vte
 
-    s = "{}.{}.{}".format(
-        Vte.MAJOR_VERSION,
-        Vte.MINOR_VERSION,
-        Vte.MICRO_VERSION,
-    )
+    s = f"{Vte.MAJOR_VERSION}.{Vte.MINOR_VERSION}.{Vte.MICRO_VERSION}"
```
Comment (mlouielu) [inline guake/__init__.py:42]: Not sure why this changed, as this is update at #1880, but in this PR didn't touch black

Comment (Davidy22) [reply guake/__init__.py:42]: The first commit with the failing build contains all the messages explaining why the build failed, the update to flakehell namely brought with it a strong preference for f strings

Comment (mlouielu) [reply guake/__init__.py:42]: Great to have them to be f-string!

**Comment by mlouielu** [comment]
Prefer to have `<verb> <description>` in commit message, e.g.

```
$ Replace dephell to flakehell in dev-packages
$ Update string format to f-string format
```

**Review by mlouielu — APPROVED** [review]
LGTM

**Pull request merged**